docs/spec/spec.md

```diff
@@ -197,52 +259,158 @@ channel._
+| `ref`<sup>1</sup> | [`duckv1.KReference`](#duckv1.kreference) | Reference to an [`duckv1.Addressable`](#duckv1.addressable).                                         | Must adhere to [`duckv1.Addressable`](duckv1.addressable). |
+| `uri`<sup>1</sup> | [`apis.URL`](#apis.url)                   | Either an absolute URL (if ref is not specified). Resolved using the base URI from ref if specified. | Must be an URL.                                            |
 
 1: One or both (ref, uri), Required. If only uri is specified, it must be an
 absolute URL. If both are specified, uri will be resolved using the base URI
 retrieved from ref.
 
-### duck.DeliverySpec
+### eventingduckv1.DeliverySpec
+
+`DeliverySpec` contains the delivery options for event senders.
 
-| Field          | Type          | Description                                                                                                                                      | Constraints |
-| -------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ | ----------- |
-| deadLetterSink | Destination   | DeadLetterSink is the sink receiving event that could not be sent to a destination.                                                              |             |
-| retry          | String        | Retry is the minimum number of retries the sender should attempt when sending an event before moving it to the dead letter sink.                 |             |
-| backoffPolicy  | BackoffPolicy | BackoffPolicy is the retry backoff policy (linear, exponential).                                                                                 |             |
-| backoffDelay   | String        | For linear policy, backoff delay is backoffDelay\*<numberOfRetries>. For exponential policy, backoff delay is backoffDelay\*2^<numberOfRetries>. |             |
+| Field Name       | Field Type                                  | Requirement | Description                                                                                                                                      | Constraints |
+| ---------------- | ------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------ | ----------- |
+| `deadLetterSink` | [`duckv1.Destination`](#duckv1.Destination) | Optional    | The sink receiving event that could not be sent to a `Destination`.                                                                              |             |
+| `retry`          | `int`                                       | Optional    | The minimum number of retries the sender should attempt when sending an event before moving it to the dead letter sink.                          |             |
```

Comment:
if there's no dead letter sink, it gets dropped? I wonder if we should say something like:
... moving it to the dead letter sink if specified, or discarded if not.

Reply:
Updated.